t/t7300-clean.sh

```diff
@@ -799,4 +799,26 @@ test_expect_success MINGW 'clean does not traverse mount points' '
 	test_path_is_file target/dont-clean-me
 '
 
+test_expect_success MINGW 'clean handles recursive symlink' '
+	rm -fr repo &&
+	mkdir repo &&
+	(
+		cd repo &&
+		git init &&
+		mkdir -p packages/some-package/node_modules &&
+		cd packages/some-package &&
+		touch package.json &&
+		git add package.json &&
+		git commit -m setup &&
+		cd node_modules &&
+		cmd //c "mklink /D /J some-package .." &&
```

Comment:
Unless I'm missreading the [`mklink` docs](https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/mklink) `/d` and `/j` seem mutually exclusive.

I think `mklink` just chooses whichever comes last. That would explain why you get `IO_REPARSE_TAG_MOUNT_POINT`.

Reply:
Weird; I just copied the command from the original issue without thinking. I'll try it both ways and see if I can figure out which one pnpm is definitely making.

Though, I have a suspicion that both should work the same?

Reply:
Thinking about it some more, `mklink /d /j` and `mklink /j` probably produce the same result.